app/controllers/declaration/TotalNumberOfItemsController.scala

```diff
@@ -60,12 +60,7 @@ class TotalNumberOfItemsController @Inject()(
   }
 
   private def nextPage(declarationType: DeclarationType): Mode => Call =
-    declarationType match {
-      case DeclarationType.SUPPLEMENTARY | DeclarationType.STANDARD | DeclarationType.CLEARANCE =>
-        controllers.declaration.routes.NatureOfTransactionController.displayPage
-      case DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL =>
-        controllers.declaration.routes.PreviousDocumentsController.displayPage
-    }
+    controllers.declaration.routes.TotalPackageQuantityController.displayPage
```

Comment:
If now it's just one line without logic, what do you think about inlining this?

Reply:
In next PR about skiping - it won't be one line